deployment/hasura/metadata/databases/AerieMerlin/tables/public_activity_directive_changelog.yaml

```diff
@@ -0,0 +1,32 @@
+table:
+  name: activity_directive_changelog
+  schema: public
+object_relationships:
+- name: activity_directive
+  using:
+    manual_configuration:
+      remote_table:
+        name: activity_directive
+        schema: public
+      column_mapping:
+        plan_id: plan_id
+        activity_directive_id: id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [ plan_id, name, source_scheduling_goal_id, changed_at, changed_by, start_offset, type,
+                 arguments, changed_arguments_at, activity_directive_id, revision, metadata, anchor_id, anchored_to_start]
```

Comment:
Minor, nonblocking: could you reorder the columns here so they're in the same order as on the table? It makes it easier to check that they match